service/device/bundle/internal/src/main/java/org/eclipse/kapua/service/device/management/bundle/internal/DeviceBundleManagementServiceImpl.java

```diff
@@ -170,7 +169,7 @@ public void start(KapuaId scopeId, KapuaId deviceId, String bundleId, Long timeo
         if (!responseMessage.getResponseCode().isAccepted()) {
             KapuaResponsePayload responsePayload = responseMessage.getPayload();
 
-            throw new BundleStartManagementException(responseMessage.getResponseCode(), responsePayload.getExceptionMessage(), responsePayload.getExceptionStack());
+            throw new KapuaException(BundleManagementResponseErrorCodes.BUNDLE_START_ERROR);
```

Comment:
Why removing Specific exception for a generic one with the same ErrorCode?

Reply:
@Coduz The problem is in descriptivness of the error message. Before it was "Error: INTERNAL_ERROR,null,null,[Ljava.lang.Object;@18955d4a" for unsuccessfull bundle start, now it is a generic message saying that bundle could not be started. User can still look in logs for proper error message, but since the error box in bottom right corner is too small for decent message, why wanted to change this to a generic message. Do you think that we should solve this differently? Incorporating a specific error message of error code?

Reply:
@Coduz Also there is the question of using this exception from service.device.management.bundle.internal package, which is then accessible to the GwtDeviceMenagementServiceImpl and its startBundle() method.
Should this class have access to exception from internal package? Which is then shown to the user?
it didn't look like a good solution to handle this internal exception in the KapuaExceptionHandler.
What do you think? What would be a better solution?

Reply:
@ct-ajovanovic  yeah! Right, I forgot about the fact that we still have exception classes on the `internal` part of a service... That unfortunately causes this problems... There is already a PR on this topic but there is no due date. Then is ok to leave things like they are! 

I'll approve the PR